keras_cv/models/backbones/efficientnet_lite/efficientnet_lite_backbone.py

```diff
@@ -0,0 +1,368 @@
+        scale=scale, mode="fan_out", distribution="truncated_normal"
+    )
+
+
+def round_filters(filters, depth_divisor, width_coefficient):
+    """Round number of filters based on depth multiplier."""
+    filters *= width_coefficient
+    new_filters = max(
+        depth_divisor,
+        int(filters + depth_divisor / 2) // depth_divisor * depth_divisor,
+    )
+    # Make sure that round down does not go down by more than 10%.
+    if new_filters < 0.9 * filters:
+        new_filters += depth_divisor
+    return int(new_filters)
+
+
+def round_repeats(repeats, depth_coefficient):
+    """Round number of repeats based on depth multiplier."""
+    return int(math.ceil(depth_coefficient * repeats))
+
+
+def apply_efficient_net_lite_block(
+    inputs,
+    activation="relu6",
+    dropout_rate=0.0,
+    name=None,
+    filters_in=32,
+    filters_out=16,
+    kernel_size=3,
+    strides=1,
+    expand_ratio=1,
+):
+    """An inverted residual block, without SE phase.
+
+    Args:
+        inputs: input tensor.
+        activation: activation function.
+        dropout_rate: float between 0 and 1, fraction of the input units to drop.
+        name: string, block label.
+        filters_in: integer, the number of input filters.
+        filters_out: integer, the number of output filters.
+        kernel_size: integer, the dimension of the convolution window.
+        strides: integer, the stride of the convolution.
+        expand_ratio: integer, scaling coefficient for the input filters.
+        id_skip: boolean.
+
+    Returns:
+        output tensor for the block.
+    """  # noqa: E501
+    if name is None:
+        name = f"block_{keras.backend.get_uid('block_')}_"
+
+    # Expansion phase
+    filters = filters_in * expand_ratio
+    if expand_ratio != 1:
+        x = keras.layers.Conv2D(
+            filters,
+            1,
+            padding="same",
+            use_bias=False,
+            kernel_initializer=conv_kernel_initializer(),
+            name=name + "expand_conv",
+        )(inputs)
+        x = keras.layers.BatchNormalization(
+            axis=BN_AXIS, name=name + "expand_bn"
+        )(x)
+        x = keras.layers.Activation(
+            activation, name=name + "expand_activation"
+        )(x)
+    else:
+        x = inputs
+
+    # Depthwise Convolution
+    if strides == 2:
+        x = keras.layers.ZeroPadding2D(
+            padding=utils.correct_pad_downsample(x, kernel_size),
+            name=name + "dwconv_pad",
+        )(x)
+        conv_pad = "valid"
+    else:
+        conv_pad = "same"
+    x = keras.layers.DepthwiseConv2D(
+        kernel_size,
+        strides=strides,
+        padding=conv_pad,
+        use_bias=False,
+        depthwise_initializer=conv_kernel_initializer(),
+        name=name + "dwconv",
+    )(x)
+    x = keras.layers.BatchNormalization(axis=BN_AXIS, name=name + "bn")(x)
+    x = keras.layers.Activation(activation, name=name + "activation")(x)
+
+    # Skip SE block
```

Comment:
What are all these comments about the SE block? `se_ratio` isn't even an arg here

Reply:
Yes, shouldn't be there. Removed.